### PR TITLE
feat: [sc-31841] Update Transformer to use new UGC Domain

### PR DESCRIPTION
**Pull request by petro-bishko-sigma**
Story details: https://app.shortcut.com/cheddar/story/31841

### CODE REVIEW

**Comment by petro-bishko-sigma** [comment]
@petro-bishko-sigma @zhiln pls review my PR

**Review by ignatospadov — APPROVED** [review]
LGTM @petro-bishko-sigma

File: src/mocks/html/editorjs/ampStory.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
 <em></em>
 <h2>WHAT ELSE IS HAPPENING ON THE GROUND:</h2>
 <amp-img width="600" height="338" layout="responsive"
-         src="https://images-cf.news12static.com/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp"
+         src="https://images.ctfassets.net/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp"
```
Comment (ignatospadov) [inline src/mocks/html/editorjs/ampStory.html:31]: shouldnt this be images-cf?

File: src/mocks/html/editorjs/story.html [hunk 2]
```diff
@@ -9,19 +9,19 @@
...
+  <embed class="form-embed" src="https://ugc.alticeustech.com/formviewer/display/365">
   <em></em>
 </div>
 <div class="embed" style="width: 552px;">
   <embed class="embed" height="315"
          width="552" src="https://www.facebook.com/video/embed?video_id=1566308450213846">
   <em></em>
 </div><h2>WHAT ELSE IS HAPPENING ON THE GROUND:</h2><div>
-  <img src="https://images-cf.news12static.com/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp" alt=""/>
+  <img src="https://images.ctfassets.net/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp" alt=""/>
```
Comment (ignatospadov) [inline src/mocks/html/editorjs/story.html:24]: shouldnt this be images-cf?

**Review by ignatospadov — CHANGES_REQUESTED** [review]
As discussed, we will need to switch to `ugc.prod.alticeustech.com`

File: src/mocks/html/editorjs/story.html (hunk 2 above)
Comment (petro-bishko-sigma) [reply src/mocks/html/editorjs/story.html:24]: Url like this https://images-cf.ctfassets.net/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp
doesn't work

File: src/mocks/html/editorjs/ampStory.html (hunk 1 above)
Comment (petro-bishko-sigma) [reply src/mocks/html/editorjs/ampStory.html:31]: Url like this https://images-cf.ctfassets.net/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp
doesn't work

this URL is working: https://images.ctfassets.net/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp

Comment (ignatospadov) [reply src/mocks/html/editorjs/ampStory.html:31]: https://images-cf.news12static.com/3kqcuzntcg31/cOpyGRYNloIdLilQw2niR/c18afe600895f0d9cdd7baae1653f21e/a7144bae-12d1-4fcf-bc20-3ff0b9457646.png_width_1200_amp_auto_webp

File: src/mocks/html/editorjs/story.html (hunk 2 above)
Comment (ignatospadov) [reply src/mocks/html/editorjs/story.html:24]: didnt we switch to using fastly to cache images? @petro-bishko-sigma

Comment (petro-bishko-sigma) [reply src/mocks/html/editorjs/story.html:24]: not sure, I did these chanegs a lot of time ago
should I use https://images-cf.news12static.com instead https://images.ctfassets.net?

Comment (ignatospadov) [reply src/mocks/html/editorjs/story.html:24]: I think so since we are already here anyway @petro-bishko-sigma

Comment (petro-bishko-sigma) [reply src/mocks/html/editorjs/story.html:24]: ok will do
thank you @ignatospadov

**Review by ignatospadov — APPROVED** [review]

**Pull request merged**